TfIdf_BOW_creator.py

```diff
@@ -0,0 +1,186 @@
+import csv
+import numpy as np
+from regex_extraction import word_processor
+import re 
+import enchant
+dict = enchant.Dict("en_US")
+from nltk.stem import PorterStemmer
+ps = PorterStemmer()
```

Comment:
And, please leave two lines after the imports

Reply:
Made changes